### ISSUE TITLE
Smooth transition when enabling/disabling

### ISSUE THREAD

**Issue by beerisgood** [Tyrrrz/LightBulb#1]
In f.lux we can force a smooth toogle to the current temperature change.
This would be nice for this tool too.

**Comment by Tyrrrz** [comment]
> force a smooth toogle to the current temperature change

Can you explain that in simpler terms?

**Comment by beerisgood** [comment]
For example its late in the night and you start the pc.
F.lux start and change the temperature not instant to the eyes-friendly-value. It uses a slow/smooth step by step to change the value. So your eyes see a smooth change and get not instant the "end" temperature.

**Comment by Tyrrrz** [comment]
I see what you mean now.

**Comment by Tyrrrz** [comment]
Added in new version

**Comment by beerisgood** [comment]
Now THAT was fast.
Thank you!

Good work.
